cirq-core/cirq/circuits/circuit.py

```diff
@@ -2416,7 +2425,14 @@ def _draw_moment_in_diagram(
 
     max_x = x0
     for op in non_global_ops:
-        indices = [qubit_map[q] for q in op.qubits]
+        qubits: Tuple[Any, ...] = tuple(op.qubits)
+        cbits: Tuple[Any, ...] = ()
+        if draw_cregs:
```

Comment:
I think we need to replace the `draw_cregs` flag with a list of cregs that we're drawing (which is based on which control bits occur in a circuit). Then we draw only matching measurement/control keys here. (And we should be able to extract the info from qubit_map rather than passing in a new param).